tendrl/node_agent/provisioner/ceph/manager.py

```diff
@@ -0,0 +1,49 @@
+import importlib
+import inspect
+import logging
+import os
+import re
+
+from tendrl.node_agent.provisioner.ceph import provisioner
+
+LOG = logging.getLogger(__name__)
+
+
+class ProvisioningManager(object):
+
+    def __init__(self, provisioner):
+        self.ceph_provisioner = provisioner
+        try:
+            self.load_plugins()
+        except (SyntaxError, ValueError, ImportError) as ex:
+            raise ex
+        self.plugin = None
+        self.set_plugin()
+
+    def load_plugins(self):
+        try:
+            path = os.path.dirname(os.path.abspath(__file__)) + '/plugins'
+            pkg = 'tendrl.node_agent.provisioner.ceph.plugins'
+            for py in [f[:-3] for f in os.listdir(path)
+                       if f.endswith('.py') and f != '__init__.py']:
+                plugin_name = '.'.join([pkg, py])
+                mod = importlib.import_module(plugin_name)
+                clsmembers = inspect.getmembers(mod, inspect.isclass)
+                for name, cls in clsmembers:
+                    exec("from %s import %s" % (plugin_name, name))
```

Comment:
exec is not safe, please use something like https://github.com/Tendrl/alerting/blob/5a84e02bf060a54f9826b0850f819a86af5642ef/tendrl/alerting/handlers/__init__.py#L118